spec/features/invite_existing_user_spec.rb

```diff
@@ -6,7 +6,7 @@
   let(:betty) { create(:user) }
   let(:confirmed_user) { create(:user) }
 
-  include_examples 'invite use case spy'
+  include_examples 'with invite use case spy'
```

Comment:
Firstly, I don't think this really makes sense. "include examples with invite use case spy"????

Secondly, here we are calling `include_examples` yet `'with invite use case spy'` is a `shared_context`.

I think we we should be using `include_context` rather than `include_examples`, and it seems to work.

Also when renaming it, perhaps something more descriptive like `include_context 'when sending an invite email'`